cmd/admin-handlers-users.go

```diff
@@ -1197,23 +1197,7 @@ func (a adminAPIHandlers) AccountInfoHandler(w http.ResponseWriter, r *http.Requ
 		return rd, wr
 	}
 
-	bucketStorageCache.Once.Do(func() {
-		// Set this to 10 secs since its enough, as scanner
-		// does not update the bucket usage values frequently.
-		bucketStorageCache.TTL = 10 * time.Second
-
-		// Rely on older value if usage loading fails from disk.
-		bucketStorageCache.ReturnLastGood = true
-		bucketStorageCache.NoWait = true
-		bucketStorageCache.Update = func() (DataUsageInfo, error) {
-			ctx, done := context.WithTimeout(context.Background(), 2*time.Second)
-			defer done()
-
-			return loadDataUsageFromBackend(ctx, objectAPI)
-		}
-	})
-
-	dataUsageInfo, _ := bucketStorageCache.Get()
```

Comment:
There was a double initialization that is removed here (see bucket-quota.go).

Reply:
Ah well I thought, the other init spot would always finish first, but I see this is not guaranteed. I'll bring the second init code back and try to remove it in a better way. In server-main.go we initialize many subsystems asynchronously, so perhaps there are more such situations where the subsystem starts getting used before init completes.